ledger-core/v2/runner/machinetype/init.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package machinetype
+
+// MachineType is a type of virtual machine
+type MachineType int
+
+// Real constants of MachineType
+const (
+	Unknown MachineType = iota
+	Builtin
+	LastID
+)
+
+func (m MachineType) Equal(other MachineType) bool {
```

Comment:
If this is for proto, then please use "casttype" option instead of "customtype" for fields with MachineType. And delete this method